pkg/operator/operator.go

```diff
@@ -41,6 +41,9 @@ var (
 
 const (
 	resyncPeriod = 5 * time.Minute
+
+	// see https://github.com/kubernetes/apiserver/blob/b571c70e6e823fd78910c3f5b9be895a756f4cbb/pkg/server/options/authentication.go#L239
```

Comment:
```suggestion
	// > authenticationConfigMapName is the name of ConfigMap in the kube-system namespace holding the root certificate
	// > bundle to use to verify client certificates on incoming requests before trusting usernames in headers specified
	// > by --requestheader-username-headers. This is created in the cluster by the kube-apiserver.
        // > "WARNING: generally do not depend on authorization being already done for incoming requests.")
	// see https://github.com/kubernetes/apiserver/blob/b571c70e6e823fd78910c3f5b9be895a756f4cbb/pkg/server/options/authentication.go#L239
```
Would you mind adding the comment directly?

Reply:
hm, I didn't want to refer to the comment, but the constant per se. I suggest to leave this for reference.